packages/angular/cli/commands/e2e-impl.ts

```diff
@@ -14,8 +14,19 @@ import { Schema as E2eCommandSchema } from './e2e';
 export class E2eCommand extends ArchitectCommand<E2eCommandSchema> {
   public readonly target = 'e2e';
   public readonly multiTarget = true;
+  public readonly missingTargetError = `
+Cannot find "e2e" target for the specified project.
 
-  public async run(options: E2eCommandSchema & Arguments) {
-    return this.runArchitectTarget(options);
+You should add a package that implements e2e capabilities.
+
+Here are some options that are available:
```

Comment:
Would be nice to keep the same format as the other builders ie: https://github.com/angular/angular-cli/blob/c3cc50e6b95fbbd814d752eec808ea0a0ef411c0/packages/angular/cli/commands/deploy-impl.ts#L18

Reply:
Mentioned `ng add` for WebdriverIO, removed TestCafe for now